### DISCUSSION TITLE
Image | NanoPC T6

### DISCUSSION THREAD

**Discussion by ghost** [MichaIng/DietPi#6441] [Image Request]
### Discussed in https://github.com/MichaIng/DietPi/discussions/6400

<div type='discussions-op-text'>

<sup>Originally posted by **wusimfan** May 30, 2023</sup>
### Creating an image request
Currently only the images of the smaller brothers NanoPi R6S and R6C are available for download. Not for the big brother NanoPC T6.

#### Formal device information
- Device name | NanoPC T6
- Official product URL | https://www.friendlyelec.com/index.php?route=product/product&path=69&product_id=292
- Official Wiki URL | https://wiki.friendlyelec.com/wiki/index.php/NanoPC-T6
- Image download URL | https://www.friendlyelec.com/image/cache/catalog/details/T6_03-900x630.jpg

#### Is the SBC officially supported by the Debian installer?
- no

#### If not, is a reliable 3rd party Debian image available for this SBC?
- Debian Bullseye Core 5.10 ARM64
- Debian Bullseye Desktop 5.10 ARM64
- Debian Bullseye Minimal 5.10 ARM64
Images provided by manufacturer FriendlyElec
https://drive.google.com/drive/folders/1dQXz0RSyAFQCHYsjnkPt_jhL9PnGqwot

</div>

**Comment by ghost** [comment]
I am running DietPi_NanoPiR6S-ARMv8-Bullseye.img on nanopc-t6. So far only sound and wifi dont work.

**Reply by MichaIng** [reply]
It does not have onboard WiFi, does it? Do you use an M.2 WiFi card, or a USB dongle?

**Reply by ghost** [reply]
You are right, it does not. I have ax210 m.2 card. This works in ubuntu jammy when .ucode files are copied to /lib/firmware but not in DietPi. The blutooth does not work on any os. Ethernet works fine. I just wanted you to know that there is not much choice of OS for nanopc-t6 and that this version works great.

**Comment by MichaIng** [comment]
Ah, duplicates with #6400
